serde_core/LICENSE-APACHE

```diff
@@ -0,0 +1 @@
+../LICENSE-APACHE
```

Comment:
Could you make sure this is correctly marked as a symlink? I think it isn't; GitHub's review ui normally makes that clear.